src/main/java/eod/warObject/character/concrete/red/AssaultDirector.java

```diff
@@ -0,0 +1,68 @@
+package eod.warObject.character.concrete.red;
+
+import eod.BoardPosition;
+import eod.Gameboard;
+import eod.Party;
+import eod.Player;
+import eod.card.abstraction.summon.SummonCard;
+import eod.card.concrete.summon.AssaultDirectorSummon;
+import eod.effect.EffectExecutor;
+import eod.param.PointParam;
+import eod.warObject.character.abstraction.Character;
+
+import java.awt.*;
+import java.util.ArrayList;
+
+import static eod.specifier.WarObjectSpecifier.WarObject;
+import static eod.specifier.condition.Conditions.Being;
+import static eod.specifier.condition.Conditions.OwnedBy;
+
+public class AssaultDirector extends Character {
+    public AssaultDirector(Player player) {
+        super(player, 2, 1, Party.RED);
+    }
+
+    @Override
+    public SummonCard getSummonCard() {
+        SummonCard c = new AssaultDirectorSummon();
+        c.setPlayer(player);
+        return c;
+    }
+
+    @Override
+    public String getName() {
+        return "突擊指揮";
+    }
+
+    @Override
+    public void attack(EffectExecutor executor) {
```

Comment:
`afterAttack()` will be called twice.
On line 39 and line 66

Reply:
To fix this, you can make original attack call another attack method prepared for child class, and then call afterAttack() in original attack method

Reply:
fixed, just by removing the afterattack() from Character lol
since every CanAttack will have its default attack().